helm/reana/templates/reana-message-broker.yaml

```diff
@@ -41,8 +41,15 @@ spec:
           name: tcp
         - containerPort: 15672
           name: management
+        volumeMounts:
+        - name: data
+          mountPath: /var/lib/rabbitmq/mnesia
```

Comment:
The mounting works well, but it may be confusing that `/var/reana/rabbitmq/rabbit@localhost` gets mounted into the pod as `/var/reana/rabbitmq/mnesia/rabbit@localhost`?  I'd suggest to keep the two paths exactly the same, i.e. including `mnesia` on both.